GoogleForCreators/web-stories-wp
# Editor: Fixes page menu layering

## Summary

This fixes the page menu layering to allow click-through in the empty section between quick actions and page actions when page is un-zoomed.

## User-facing changes

| Before | After |
|-|-|
| ![page-menu-layer-before](https://user-images.githubusercontent.com/637548/161126176-7f7645a4-1c21-4a49-93bc-6153273a3163.gif) | ![page-menu-layer-after](https://user-images.githubusercontent.com/637548/161126187-4d6ce9f4-5522-4a65-8e24-41fd7635f854.gif) |

## Testing Instructions



This PR can be tested by following these steps:

1. Drag an element so it's right edge handle is in line with the page menu (between quick-actions and page-actions)
2. Drag the right edge handle and observe that it is in fact draggable
3. Observe that buttons in quick actions menu and page actions menu are still clickable as well

## Checklist




---

Fixes #10441
